straxen/plugins/event_processing.py

```diff
@@ -728,35 +748,67 @@ def compute(self, events):
             s2_top_map_name = "map"
             s2_bottom_map_name = "map"
 
+        regions = {'ab': self.ab_region, 'cd': self.cd_region}
+
+        # setup SEG and EE corrections
+        if isinstance(self.se_gain, dict):
+            seg = self.se_gain
+        else:
+            seg = {key: self.se_gain for key in regions}
```

Comment:
Sorry, I am missing something here. Where is the difference between the dict and non dict case? Should not it be `self.se_gain[key]`? Otherwise  I do not see why 
```
seg_ee_corr = seg[partition]/avg_seg[partition]*ee[partition]
```
should work as seg, av_seg and ee are dicts. But since your tests are passing I am missing something somewhere.

Reply:
Same below

Reply:
If it's not a dict, it's just a single value and we use the same correction everywhere. This part of the code converts the non-dict into a dict but without the partitioned functionality.